src/pc.jl

```diff
@@ -279,11 +278,11 @@ function pcalg(t, p::Float64, test::typeof(gausscitest); kwargs...)
     c = Tables.columns(t)
     sch = Tables.schema(t)
     n = length(sch.names)
-  
-    X = reduce(hcat, map(c->Tables.getcolumn(Tables.columns(t), c), Tables.columnnames(t)))
-    N = size(X,1)
+
+    X = reduce(hcat, map(c -> Tables.getcolumn(Tables.columns(t), c), Tables.columnnames(t)))
+    N = size(X, 1)
     C = Statistics.cor(X)
-    return pcalg(n, gausscitest, (C,N), quantile(Normal(), 1-p/2); kwargs...)
+    return pcalg(n, gausscitest, (C, N), quantile(Normal(), 1 - p / 2); kwargs...)
```

Comment:
```suggestion
    return pcalg(n, gausscitest, (C, N), quantile(Normal(), 1 - p/2); kwargs...)
```
I like it if whitespace use clarifies precedence.

Reply:
This wasn't intentional - it was the "format on save" in VS Code. I can nuke all these changes when everything is ready, but it would be even better to set a consistent formatting for the package with https://github.com/domluna/JuliaFormatter.jl 

Would you have a preferred style? Blue/SciML?

Reply:
Okay, I'll add a formatter .toml file specifying [SciML style](https://github.com/SciML/SciMLStyle) and run the whole code base through it.

As for the operator precedence-based spacing, I cannot find the corresponding option [here](https://domluna.github.io/JuliaFormatter.jl/stable/) - do you know which one it would be?

Reply:
(The rationale being that whitespace around multiplications is less bad than what we have now ;-) )